### ISSUE TITLE
Enhancement: Cannot login by pressing 'enter' key

### ISSUE THREAD

**Issue by MasssiveJuice08** [Asign2/Assignment-2-Scrum-Team-Assignment#13]
If username and password are entered, user must click 'Sign In' button to login. Users should be able to press <kbd>Enter</kbd> key to submit login request for accessibility.
